tools/automatic_query_fixer/src/test/java/com/google/cloud/bigquery/utils/queryfixer/QueryTokenProcessorTest.java

```diff
@@ -10,21 +11,20 @@
 
 public class QueryTokenProcessorTest {
 
-  private QueryTokenProcessor createService() {
-    return new QueryTokenProcessor(new BigQueryParserFactory());
+  private QueryTokenProcessor tokenService;
+
+  @Before
```

Comment:
Since we only need this once, you can use @BeforeClass?

Reply:
Good point

Reply:
I think `@Before` is better because I need it to init a member variable. `@BeforeClass` is a static method and cannot init a member variable.